pkgs/top-level/aliases.nix

```
@@ -469,6 +469,7 @@ mapAliases {
   fastnlo = throw "'fastnlo' has been renamed to/replaced by 'fastnlo-toolkit'"; # Converted to throw 2024-10-17
   fastnlo_toolkit = fastnlo-toolkit; # Added 2024-01-03
   fcitx5-catppuccin = catppuccin-fcitx5; # Added 2024-06-19
+  fdr = throw "fdr cannot be built from source and depends on Python 2.x"; # Added 2025-03-19
```

```suggestion
  fdr = throw "fdr has been removed, as it cannot be built from source and depends on Python 2.x"; # Added 2025-03-19
```
nit: maybe be explicit about the removal?

---

Sure, I can do that